src/__experimental__/components/simple-color-picker/color-option/style/color-option-classic.style.js

```diff
@@ -0,0 +1,8 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../../../style/themes';
+
+export default ({ theme }) => theme.name === THEMES.classic
+  && css`
+    margin-right: 1px;
```

Comment:
If there are only two rules, maybe it would be good to keep it in original file? (color-option)